test/Microsoft.ML.Tests/Transformers/WordTokenizeTests.cs

```diff
@@ -65,6 +78,24 @@ public void TestOldSavingAndLoading()
                     new WordTokenizingTransformer.ColumnInfo("B", "TokenizeB"),
                 });
             var result = pipe.Fit(dataView).Transform(dataView);
+
+            // Extract the transformed result of the first row (the only row we have because data contains only one TestClass) as a native class.
+            var nativeResult = new List<NativeResult>(result.AsEnumerable<NativeResult>(Env, false))[0];
+
+            // Check the tokenization of A. Expected result is { "This", "is", "a", "good", "sentence." }.
+            var tokenizeA = new[] { "This", "is", "a", "good", "sentence." };
```

Comment:
>tokenizeA [](start = 16, length = 9)

it's all cool and nice, but I would rather prefer you to add proper example to [https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic)
than modifying this test.

Reply:
We still need a test to test "end-to-end" in terms of C# native data strucutre.

Reply:
Sure.
But this test is more about saving/loading in old way, can you move code to Workout test?

And in #1746 you state what you were looking for example, and we usually have examples in Samples project, can you add it to it?

---
In reply to: [237256819](https://github.com/dotnet/machinelearning/pull/1747#discussion_r237256819) [](ancestors = 237256819)

Reply:
+1

---
In reply to: [237193355](https://github.com/dotnet/machinelearning/pull/1747#discussion_r237193355) [](ancestors = 237193355)

Reply:
I can but not in this PR. This PR is super low priority.